WordPress/Classes/ViewRelated/System/WPTabBarController.m

```diff
@@ -154,36 +151,27 @@ - (void)decodeRestorableStateWithCoder:(NSCoder *)coder
     [super decodeRestorableStateWithCoder:coder];
 }
 
-#pragma mark - Tab Bar Items
+#pragma mark - My Site View Controllers
 
-- (UINavigationController *)blogListNavigationController
-{
-    if (_blogListNavigationController) {
-        return _blogListNavigationController;
-    }
+// These getters are temporary while we extract items from here to the MySiteCoordinator.
```

Comment:
Just a note that I'll be removing these in a follow-up PR, which will move the navigation methods that rely on these properties. I figured this PR was big enough already :)

Reply:
👍